### PR TITLE
Add limitations for multi-cluster concept

### PR DESCRIPTION
**Pull request by subhankarc**

### CODE REVIEW

**Comment by subhankarc** [comment]
@amshuman-kr @PadmaB Can you have a glance please ?

File: inter-operator/architecture/basic.md [hunk 1]
```diff
@@ -617,3 +618,8 @@ Now the setup is ready for taking requests. We depict this in the picture below.
...
 ![Inter-operator Runtime Flow](images/Runtime%20Flow%20Updated.png)
+
+## Limitations with Multi-Cluster deployment
+1. Interoperator currently does not take care of the cluster off-boarding.
+2. Service Operator in each sister cluster is assumed to be already deployed and version update-upgrade is managed/maintained by the service operator. Inter-operator does not do anything about it.
```
Comment (amshuman-kr) [inline inter-operator/architecture/basic.md]:
```suggestion
2. Service Operator in each sister cluster is assumed to be already deployed and its version update/upgrade is managed/maintained by the service operator. Inter-operator does not do anything about it.
3. Interoperator does not take care of the Kubernetes and OS updates to the onboarded clusters.

Comment (PadmaB) [reply inter-operator/architecture/basic.md]: Also makes sense to add the below point, 
Service owners will have to monitor the clusters and their resource situations and add additional sister clusters if required

Comment (subhankarc) [reply inter-operator/architecture/basic.md]: Incorporated both the suggestions. Thanks @amshuman-kr @PadmaB !

**Pull request merged**